File: G50/Sebastian_Rojas_Vargas/Diagnostico1/PunterosFutbolJava/src/Main/Jugador.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Main;

/**
 *
 * @author Sebas
 */
public class Jugador {

    String nombre;
    int numeroCamisa;
    Jugador  sig, ant;
    
    public Jugador(String nombre, int numeroCamisa) {
        this.nombre = nombre;
        this.numeroCamisa = numeroCamisa;
        this.sig = this.ant = null;
    }

}

File: G50/Sebastian_Rojas_Vargas/Diagnostico1/FutboPunterosPython/Jugador.py
class Jugador():
    def __init__(self):
        self.nombre = ""
        self.numeroCamisa = 0
        self.ant = None
        self.sig = None
    def set__info(self, nombre, numeroCamisa):
        self.nombre = nombre
        self.numeroCamisa = numeroCamisa
    def get__info(self):
        print (self.nombre, self.numeroCamisa)


